### PR TITLE
Static Behavior on serialize object

### PR DESCRIPTION
**Pull request by bloodbare**

### CODE REVIEW

File: guillotina/json/serialize_content.py [hunk 1]
```diff
@@ -56,20 +56,24 @@ def __init__(self, context, request):
...
+            behaviors.append(behavior_schema)
+
         result = {
             '@id': IAbsoluteURL(self.context, self.request)(),
             '@type': self.context.type_name,
             '@name': self.context.__name__,
             '@uid': self.context.uuid,
             'parent': parent_summary,
             'is_folderish': IFolder.providedBy(self.context),
+            '__static_behaviors__': behaviors,
```
Comment (vangheem) [inline guillotina/json/serialize_content.py]: Can we use a `@` value here?

Sorry, I don't like adding non-editable attributes without the `@`.

File: guillotina/json/serialize_content.py [hunk 2]
```diff
@@ -56,20 +56,24 @@ def __init__(self, context, request):
...
+        behaviors = []
+        for behavior_schema in factory.behaviors or ():
+            behaviors.append(behavior_schema)
+
         result = {
             '@id': IAbsoluteURL(self.context, self.request)(),
             '@type': self.context.type_name,
             '@name': self.context.__name__,
             '@uid': self.context.uuid,
+            '@sb': behaviors,
```
Comment (vangheem) [inline guillotina/json/serialize_content.py]: sorry, can we just call it "@static_behaviors"?

Comment (bloodbare) [reply guillotina/json/serialize_content.py]: this information is going to be on all responses of GET so I was willing to keep it short

Comment (vangheem) [reply guillotina/json/serialize_content.py]: eh, I don't know man. I think we need the name to be descriptive.

Up to you now. I'll approve but you're free to make final decision.

**Pull request merged**